Changes to GEOMAR boot camp page.

amyrhoda opened the pull request:
Removed reference to _content.html, added HTML for their planned
syllabus. Also added Germany to location because I didn't know where
Kiel was. (Kept wanting it to be typo of Kiev.)

amyrhoda committed:
Changed content description for GEOMAR page

Removed reference to _content.html, added HTML for their planned
syllabus. Also added ,Germany to location because I didn't know where
Kiel was. (Kept wanting it to be typo of Kiev.)

gvwilson commented on bootcamps/2013-05-geomar.html:
```diff
@@ -7,11 +7,70 @@
+</p>
+<ul>
+  <li>Unix shell</li>
+  <ul>
```
Strictly speaking, you can't put a `<ul>` directly in a `<ul>`: it has to be inside one of the `<li>`'s, like this:

``` html
<ul>
  <li>First major
    <ul>
      <li>First minor</li>
      <li>Second minor</li>
    </ul>
  </li>
  <li>Second major
    <ul>
      <li>Third minor</li>
    </ul>
  </li>
</ul>
```

amyrhoda replied:
Huh. I tried that and it looks like this: 
![dbleList](https://f.cloud.github.com/assets/3077295/112267/748ef24a-6b16-11e2-88c6-5ed6e66ce721.jpg)

Note goofy double bullets. Am I doing something wrong? Here's my HTML:

```
On the second day we will cover:
</p>
<ul>
  <li>Python</li>
  <li>
    <ul>
      <li>Finding help</li>
      <li>Starting a script</li>
```

gvwilson replied:
On 2013-01-30 2:52 PM, Amy Brown wrote:

> In bootcamps/2013-05-geomar.html:
> 
> >  {% include "_requirements.html" %}
> > -{% include "_content.html" %}
> > +
> > +<p>
> > -  <strong>Content:</strong> On the first day of the bootcamp we will cover:
> >   +</p>
> >   +<ul>
> > -  <li>Unix shell</li>
> > -  <ul>
> 
> Huh. I tried that and it looks like this:
> dbleList 
> https://f.cloud.github.com/assets/3077295/112267/748ef24a-6b16-11e2-88c6-5ed6e66ce721.jpg
> 
> Note goofy double bullets. Am I doing something wrong? Here's my HTML:
> 
> |On the second day we will cover:
> </p>
> <ul>
>    <li>Python</li>
>    <li>
>      <ul>
>        <li>Finding help</li>
>        <li>Starting a script</li>
> |
> 
> What you have created there is:
> - one bullet saying "Python"
> - a separate bullet containing a sub-list of two bullets "Finding help" 
>   and "Starting a script"
>   What this should be is:
>   <ul>
>   <li>Python <!-- with no closing </li> -->
>    <ul> <!-- inside the first 'li' of the outer list -->
>      <li>Finding help</li> <!-- becomes the first bullet of the 
>   sublist of the first major point -->
>      <li>Starting a script</li>
>    </ul> <!-- closes the sublist -->
>   </li> <!-- closes the first point of the outer list -->
>   <li>Confusion</li> <!-- a bullet point in the outermost list that 
>   doesn't contain a sub-list, just for example -->
>   </ul> <!-- close the major list -->

jiffyclub replied:
You want this:

``` html
<ul>
  <li>Python
    <ul>
      <li>Finding help</li>
      <li>Starting a script</li>
   </ul>
  </li>
</ul>
```

Which renders to:

<ul>
  <li>Python
    <ul>
      <li>Finding help</li>
      <li>Starting a script</li>
   </ul>
  </li>
</ul>


Note where the closing `</li>` element occurs for the Python item.

amyrhoda commented:
Oooh! Duh. Thanks.

amyrhoda committed:
Fixed broken list

I moved the sublist into the same <li> element as the headers in the
list of topics, for more correctness.
